tools/chrome-debug-extension/src/telemetryViewerPopup.tsx

```diff
@@ -72,7 +76,43 @@ export const TelemetryViewerPopup = (): React.ReactElement => {
         }
     }
 
+    function highlightNewVersion() {
+        let orgTitle = document.title;
+        let count = 0;
+        let interval = setInterval(() => {
+            count ++;
+            if ((count % 2) == 0) {
+                document.title = orgTitle;
+            } else {
+                document.title = orgTitle + " *** v" + encodeURIComponent(newAvailableVersion) + " Available ***"
```

Comment:
Very simple "flashing" title to show the new available version number (really just a different version -- assumes that we only move forward)